packages/gatsby-theme-docs/gatsby-node.js

```diff
@@ -147,31 +161,58 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
   if (navigationYamlResult.errors) {
     reporter.panicOnBuild('🚨  ERROR: Loading "allNavigationYaml" query');
   }
-  const pages = allMdxPagesResult.data.allMdx.edges;
+  const pages = allMdxPagesResult.data.contents.nodes;
   const navigationPages = navigationYamlResult.data.allNavigationYaml.nodes.reduce(
     (pageLinks, node) => [...pageLinks, ...(node.pages || [])],
     []
   );
-  pages.forEach(({ node }) => {
-    const matchingNavigationPage = navigationPages.find(
-      (page) =>
-        trimTrailingSlash(page.path) === trimTrailingSlash(node.fields.slug)
-    );
-    actions.createPage({
-      // This is the slug you created before
-      // (or `node.frontmatter.slug`)
-      path: node.fields.slug,
-      // This component will wrap our MDX content
-      component: require.resolve('./src/templates/page-content.js'),
-      // You can use the values in this context in
-      // our page layout component
-      context: {
-        ...node.fields,
-        shortTitle: matchingNavigationPage
-          ? matchingNavigationPage.title
-          : undefined,
-      },
-    });
+  pages.forEach(({ childMdx }) => {
+    if (childMdx) {
+      const matchingNavigationPage = navigationPages.find(
+        (page) =>
+          trimTrailingSlash(page.path) ===
+          trimTrailingSlash(childMdx.fields.slug)
+      );
+      actions.createPage({
+        // This is the slug you created before
+        // (or `node.frontmatter.slug`)
+        path: childMdx.fields.slug,
+        // This component will wrap our MDX content
+        component: require.resolve('./src/templates/page-content.js'),
+        // You can use the values in this context in
+        // our page layout component
+        context: {
+          ...childMdx.fields,
+          shortTitle: matchingNavigationPage
+            ? matchingNavigationPage.title
+            : undefined,
+        },
+      });
+    }
+  });
+
+  // create release notes overview page
+  actions.createPage({
+    path: '/release-notes',
+    component: require.resolve('./src/templates/release-notes.js'),
+  });
+
+  // create details pages for each release note
+  allMdxPagesResult.data.releaseNotes.nodes.forEach(({ childMdx }) => {
+    if (childMdx) {
+      actions.createPage({
+        // This is the slug you created before
+        // (or `node.frontmatter.slug`)
+        path: childMdx.fields.slug,
```

Comment:
@nkuehn was mentioning something about the URLs of the release notes to be different than the content pages...not sure what the expected outcome here is.